bin/jobsub_fetchlog

```diff
@@ -51,6 +54,145 @@ from version import print_version, print_support_email
 
 verbose = 0
 
+# environment variable containing base fetchlog server url
+_FETCHLOG_URL_ENV = "JOBSUB_FETCHLOG_URL"
```

Comment:
Personally, I would like to see this defined much closer to where it gets used; like down near line 132.  But don't hold up on merging for this...

Reply:
We can certainly have that discussion.  I tend to do the same as Kevin has here - all constants at the top of the module, but I do see merits to defining it right in the function, especially if it's only going to be used there.  For now, I'll merge, as you said, Marc, but perhaps this discussion can take place in a separate issue or even offline?

Reply:
Yeah I prefer constants at the top as well, collected together and documented. Or even in their own file.  I've also introduced a new constant type here, the "env var name", which we have hardcoded elsewhere. I'd like to see that more uniform, to make it easier to know what env vars the code uses, and make sure the config aligns.